### PR TITLE
Release 1.8.1-snapshot.20210115.5842.0.59f5d407

### PR DESCRIPTION
**Pull request by fabiotudone-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by gerolf-da — APPROVED** [review]

File: LATEST [hunk 1]
```diff
@@ -1,5 +1,6 @@
 9ed787cb3dadd2d6d79f77a2ee93c576f8b8ac4c 1.9.0-snapshot.20210113.6060.0.9ed787cb
 a8ae8e4aa782b8949d32aec3ae05a3211a2306f7 1.8.0-snapshot.20201214.5841.0.a8ae8e4a
```
Comment (cocreature) [inline LATEST:2]: Can we replace this 1.8.0 snapshot? I forgot what it was for but having to 1.8.0, 1.8.0 snapshot and 1.8.1 snapshot seems off.

Comment (fabiotudone-da) [reply LATEST:2]: I don' think we can; @gerolf-da?

Comment (cocreature) [reply LATEST:2]: ah right that was the fast but broken version. Alright, then let’s keep it. There is nothing stopping us from deleting that line later once vmbc doesn’t need it anymore.

**Comment by fabiotudone-da** [comment]
Manual tests passed on Windows.

**Comment by fabiotudone-da** [comment]
Manual tests passed on Linux.

**Pull request merged**